debugger/ptraceLib/sgxtypes.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_SGX_TYPES_H
```

Comment:
Can this just be `_SGXTYPES_H` to match the filename?

Reply:
Fixed.